lib/elixir/test/elixir/option_parser_test.exs

```diff
@@ -256,6 +256,12 @@ defmodule OptionParserTest do
            == {[foo: true, boo: "-"], ["-"], []}
   end
 
+  test "allow nonexistent atom`s" do
+    opts = [switches: [a: :string], allow_nonexistent_atoms: true]
+    assert OptionParser.parse(["--a", "b", "--c", "d", "--option-key-creates-atom", "f"], opts)
```

Comment:
We usually put the operator at the end of the line, so I would move the `==` at the end of the previous line.